ui/css/treeherder.css

```diff
@@ -1421,144 +939,6 @@ fieldset[disabled] .btn-resultset:hover {
-  color: white;
-}
-.btn-dkgray.disabled:hover,
-.btn-dkgray.disabled:focus,
-.btn-dkgray.disabled:active,
-.btn-dkgray.disabled.active,
-.btn-dkgray[disabled]:hover,
-.btn-dkgray[disabled]:focus,
-.btn-dkgray[disabled]:active,
-.btn-dkgray[disabled].active,
-fieldset[disabled] .btn-dkgray:hover,
-fieldset[disabled] .btn-dkgray:focus,
-fieldset[disabled] .btn-dkgray:active,
-fieldset[disabled] .btn-dkgray.active {
-  background-color: #7c7a7d;
-  border-color: #7c7a7d;
-  color: white;
-}
-
-.btn-yellow,
-.btn-yellow-count,
-.btn-yellow-classified,
-.btn-yellow-classified-count {
-  background-color: #fcf8e3;
-  border-color: #fbd890;
-  color: #8a6d3b;
-  font-weight: bold;
-}
-.btn-yellow-count:hover,
-.btn-yellow-classified-count:hover {
-  color: #8a6d3b;
-}
-.btn-yellow:hover,
-.btn-yellow-classified:hover,
-.btn-yellow:focus,
-.btn-yellow:active,
-.btn-yellow.active {
-  background-color: #8a6d3b;
-  border-color: #fbd890;
-  color: #fcf8e3;
-}
-.btn-yellow.disabled:hover,
-.btn-yellow.disabled:focus,
-.btn-yellow.disabled:active,
-.btn-yellow.disabled.active,
-.btn-yellow[disabled]:hover,
-.btn-yellow[disabled]:focus,
-.btn-yellow[disabled]:active,
-.btn-yellow[disabled].active,
-fieldset[disabled] .btn-yellow:hover,
-fieldset[disabled] .btn-yellow:focus,
-fieldset[disabled] .btn-yellow:active,
-fieldset[disabled] .btn-yellow.active {
-  background-color: #fcf8e3;
-  border-color: #fbd890;
-}
-
-.btn-pink,
-.btn-pink-count,
-.btn-pink-classified,
-.btn-pink-classified-count,
-.btn-pink-count:hover,
-.btn-pink-classified-count:hover {
-  color: #ff40d9;
-  font-weight: bold;
-}
-.btn-pink:hover,
-.btn-pink-classified:hover,
-.btn-pink:focus,
-.btn-pink:active,
-.btn-pink.active {
-  background-color: #ff40d9;
-  border-color: #ff40d9;
-  color: white;
-}
-.btn-pink.disabled:hover,
-.btn-pink.disabled:focus,
-.btn-pink.disabled:active,
-.btn-pink.disabled.active,
-.btn-pink[disabled]:hover,
-.btn-pink[disabled]:focus,
-.btn-pink[disabled]:active,
-.btn-pink[disabled].active,
-fieldset[disabled] .btn-pink:hover,
-fieldset[disabled] .btn-pink:focus,
-fieldset[disabled] .btn-pink:active,
-fieldset[disabled] .btn-pink.active {
-  background-color: #ff80e5;
-  border-color: #ff80e5;
-}
-
-.btn-repo {
-  border-color: #c3c3c3;
-  color: #7c7a7d;
-  background-color: #eaeaea;
-  border-bottom-left-radius: 0;
-  border-top-left-radius: 0;
-}
-.btn-repo:hover,
-.btn-repo:focus,
-.btn-repo:active,
-.btn-repo.active {
-  background-color: #6f6d70;
-  border-color: #626163;
-  color: white;
-}
-.btn-repo.disabled:hover,
-.btn-repo.disabled:focus,
-.btn-repo.disabled:active,
-.btn-repo.disabled.active,
-.btn-repo[disabled]:hover,
-.btn-repo[disabled]:focus,
-.btn-repo[disabled]:active,
-.btn-repo[disabled].active,
-fieldset[disabled] .btn-repo:hover,
-fieldset[disabled] .btn-repo:focus,
-fieldset[disabled] .btn-repo:active,
-fieldset[disabled] .btn-repo.active {
-  background-color: #7c7a7d;
-  border-color: #7c7a7d;
-  color: white;
-}
```

Comment:
This `.btn-repo` class was created in Feb, 14' in https://github.com/mozilla/treeherder/commit/b2ba827a28aaefe653d6b0b538d3339dddfcfb29 and appears to be unused.